react/features/recording/components/RecordingLabel.native.js

```diff
@@ -41,8 +40,20 @@ class RecordingLabel extends AbstractRecordingLabel<Props> {
             return null;
         }
 
+        let extraBehaviour;
+
+        switch (this.props._status) {
+        case JitsiRecordingConstants.status.PENDING:
+            extraBehaviour = CircularLabel.behaviour.IN_PROGRESS;
```

Comment:
I think this 2 could be baked into the circular label as some `mode` prop, WDYT?

Reply:
Not sure what you mean. This controls a prop in the circular label. Or how do you mean?